CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Master
+
+##### Bug Fixes
+
+* Make linting faster than 0.5.0, but slower than 0.4.0  
+  [Norio Nomura](https://github.com/norio-nomura)
+  [#263](https://github.com/realm/SwiftLint/pull/263)
```

Comment:
please refer to issues fixed rather than this PR's number here, as outlined in CONTRIBUTING.md.